src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,8 @@
-const numberToReversedDigits = number => {};
+const numberToReversedDigits = number => {
+  let array = [];
+  return Array.from(number.toString())
```

Comment:
Based on your approach, you no longer need that empty array you defined on line 2.
Also, for the future, don't call array to a variable, give it a name that is a bit more descriptive :}